revscoring/datasources/meta/filters.py

```diff
@@ -100,3 +100,23 @@ def __init__(self, numbers_datasource, name=None):
 
     def is_negative(self, v):
         return v < 0
+
+
+class not_none(filter):
+    """
+    Generates a filtered list of not None entries from a list.
+
+    :Parameters:
+        items_datasource : :class:`revscoring.Datasource`
+            A datasource that generates the subset of items that are None
+        name : `str`
+            A name for the datasource.
+    """
+
+    def __init__(self, items_datasource, name=None):
+        name = self.format_name(name, [items_datasource])
+        super().__init__(self.is_not_none, items_datasource, name=name)
+
+    def is_not_none(self, v):
+        if v is not None:
+            return v
```

Comment:
This should return a boolean.  I think you just want to return `v is not None`

Reply:
Wait, it returns v if v is not None

if we return v is not None, that will be the boolean right?
Please confirm